heft-plugins/heft-jest-plugin/src/index.ts

```diff
@@ -2,15 +2,9 @@
 // See LICENSE in the project root for license information.
 
 /**
- * A Heft plugin for using Jest during the "test" stage.
+ * A Heft plugin for using Jest.
  *
  * @packageDocumentation
  */
 
-import type { IHeftPlugin } from '@rushstack/heft';
-import { JestPlugin } from './JestPlugin';
-
-/**
- * @internal
- */
-export default new JestPlugin() as IHeftPlugin;
+export {};
```

Comment:
Why even bother having an index.ts file if this project doesn't export anything?

Reply:
I don't mind removing the `index.ts`, just thought that it allows us to leave the door open for exporting things later on if necessary.